src/test/java/com/code_intelligence/jazzer/mutation/mutator/StressTest.java

```diff
@@ -371,6 +409,9 @@ private static <T> void testReadWriteExclusiveRoundtrip(Serializer<T> serializer
 
   private static <T> void testReadWriteRoundtrip(Serializer<T> serializer, T value)
       throws IOException {
+    // System.out.println(value.toString());
+    value = fixFloatingPointsForProtos(
```

Comment:
I admire the level of hackery that goes went into this workaround, but I would be more comfortable with an approach that makes the equality assert below more lenient. Maybe we could use https://truth.dev/api/latest/com/google/common/truth/extensions/proto/ProtoSubject.html#usingDoubleTolerance(double) with a very low tolerenace (maybe even 0?).